meson.build

```diff
@@ -214,6 +214,12 @@ else
   rizin_bindings = rizin_libdir / 'rizin-bindings'
 endif
 
+# Calcualte BINDIR's depth to be able to find the root directory during runtime
+# in portable builds
+py_cmd = 'import os; print(len(os.path.normpath(r"@0@").split(os.sep)) - len(os.path.normpath(r"@1@").split(os.sep)))'.format(rizin_prefix / rizin_bindir, rizin_prefix)
+py_cmd = run_command(py3_exe, '-c', py_cmd)
+bindir_depth = py_cmd.stdout().strip()
+
```

Comment:
bindir_depth should be somehow used as well in :
```
if use_rpath
  rpath_exe = '$ORIGIN/../' + get_option('libdir')
  rpath_lib = '$ORIGIN'
  rpath_summary = 'relative'
elif use_rpath_absolute
```

As you can see it assumes that the binaries are in `<top-prefix>/bin`, but this is not true. That should be fixed as well to ensure proper rpath.

Reply:
I had to set the minimum meson version to 0.58.0 for ranges() support